### ISSUE TITLE
ssid umbenennung 

### ISSUE THREAD

**Issue by flemming-pr** [freifunk-flensburg/Voting#10]
es wurde vorgeschlagen, dass wir im 5ghz band die ssid wieder in flensburg.freifunk.net (5ghz) umbenennen, damit man sie genauer aussuchen kann mit welchen netz man sich verbindet und auch weil einige clients damit wohl nicht ganz zurecht kommen wenn zwei netze gleich heißen usw.

**Comment by horstfffl** [comment]
hat das irgendwelche nachteile? mir fällt grad nix ein

**Comment by muellermartin** [comment]
Ich würde den Text in den Klammern aber `5 GHz` schreiben, da man üblicherweise Einheiten durch Leerzeichen von Zahlenwert trennt und man die Einheit Hertz mit `Hz` und die SI-Präfix für Giga mit `G` abkürzt.

**Comment by spralexx** [comment]
Back to "(5 Ghz)"

**Comment by roikiermedia** [comment]
+1 "freifunk.flensburg.net (5GHz)"

**Comment by foertel** [comment]
~0 mir ist das egal

Wenn sich nun einer mit freifunk flensburg 5 ghz verbindet, wird er später keine Verbindung bekommen, wenn dort nur 2,4 GHz verfügbar ist. Fügt er beide WLANs hinzu, dann wird sich sein Handy eh wieder mit irgendeinem von beiden verbinden und der Vorteil der getrennten Namen ist dahin.

So oder so bleibt sich das alles völlig gleich. ;-)

**Comment by S4ra** [comment]
Ich stimme Martins Argumentation voll und ganz zu, also +1 "freifunk.flensburg.net (5 GHz)"

**Comment by foertel** [comment]
@S4ra Martin hat ja nur argumentiert, wie man "5 Ghz" richtig schreibt, aber nicht gevotet, ob man es überhaupt tun sollte.

**Comment by bigfootDK** [comment]
-1
IMHO soll das netz nur flensburg.freifunk.net heissen - egal auf welcher funk freq. Sonst gäbe es dann mal freifunk.flensburg.net (433MHz)(2.4GHz)(5GHz)(24 GHz) - das würde doch zu weit gehen ?

**Comment by S4ra** [comment]
@foertel: Ich wollte auch nur zum Ausdruck bringen, dass ich seine Argumentation gut fand :dancers:
Also ich bleib bei meiner Stimme und @muellermartin kann sich dann ja noch äußern oder auch nicht :blush:

**Comment by horstfffl** [comment]
ich enthalte mich

**Comment by flemming-pr** [comment]
+1

**Comment by Wlanfr3ak** [comment]
+1

Begründung:
Durch zwei Namen kann zwischen den Frequenzen und auch damit den Geschwindigkeiten klar entschieden werden. bsp. im CoWork ist ein akzeptabler Downloadspeed nur über 5 GHz möglich da das 2,4 GHz Band dort absolut dicht ist. Wäre es der gleiche Name müsste dies umständlich konfiguriert werden und evtl. für Anfänger nicht einfach. Natürlich gibt es dadurch eine Auswahl mehr, ich gehe davon aus das Leute die das WLAN anklicken können das auch nutzen könnten. Sollte dann einmal das 5 GHz nicht verfügbar sein wäre zumindest das 2,4 GHz zum anklicken verfügbar = zumutbar.
